<?xml version="1.0"?>

<launch>

    <!-- <include file="$(find dynamic_obstacle_avoidance_planner)/launch/diff_drive_sim.launch"/> -->
    <node pkg="dynamic_obstacle_avoidance_planner" type="sfm_obstacle_simulator" name="sfm_obstacle_simulator">
        <param name="SIMULATION_SQUARE_LENGTH" value="20"/>
        <param name="DESIRED_FORCE_FACTOR" value="1.0"/>
        <param name="SOCIAL_FORCE_FACTOR" value="20.0"/>
        <param name="NUM_OBSTACLE" value="30"/>
    </node>
    <include file="$(find dynamic_obstacle_avoidance_planner)/launch/tf_to_obstacles.launch">
        <arg name="noise_enabled" value="true"/>
    </include>
    <node pkg="dynamic_obstacle_avoidance_planner" type="sim_3dof.py" name="sim_3dof">
        <param name="INIT_X" value="-10.0"/>
        <param name="INIT_Y" value="0.0"/>
        <param name="INIT_YAW" value="0.0"/>
    </node>
    <node pkg="dynamic_obstacle_avoidance_planner" type="waypoints_test" name="waypoints_test"/>
    <node pkg="dynamic_obstacle_avoidance_planner" type="trajectory_logger" name="trajectory_logger"/>
    <node pkg="dynamic_obstacle_avoidance_planner" type="measurer" name="measurer" output="screen"/>
    <node pkg="tf" type="static_transform_publisher" name="map_to_odom_tf_publisher" args="0 0 0 0 0 0 /map /odom 10"/>

    <include file="$(find dynamic_obstacle_avoidance_planner)/launch/collision_detector.launch"/>

</launch>
